src/main/java/edu/ucsb/cs156/organic/controllers/ApiController.java

```diff
@@ -55,6 +52,21 @@ public Object handleGenericException(Throwable e) {
     );
   }
 
+  /**
```

Comment:
See, here you added a javadoc comment.  That's good.  So, let's be consistent and keep the other one as well.